src/extractors/extractEventDefinitionVariables.js

```diff
@@ -0,0 +1,70 @@
+import {
+  forEach,
+  isArray
+} from 'min-dash';
+
+import {
+  getErrorEventDefinitions,
+  getEscalationEventDefinitions
+} from '../util/EventDefinitionsUtil';
+
+import {
+  createProcessVariable,
+  addVariableToList
+} from '../util/ProcessVariablesUtil';
+
+
+export default function(options) {
+  var elements = options.elements,
+      containerElement = options.containerElement,
+      processVariables = options.processVariables;
+
+  var addVariable = function(element, name) {
+    var newVariable = createProcessVariable(
+      element,
+      name,
+      containerElement
+    );
+
+    addVariableToList(processVariables, newVariable);
+  };
+
+  if (!isArray(elements)) {
+    elements = [ elements ];
+  }
+
+  forEach(elements, function(element) {
+
+    // (1) error event code + message variable
+    var errorEventDefinitions = getErrorEventDefinitions(element);
+
+    forEach(errorEventDefinitions, function(definition) {
+
+      var errorCodeVariable = definition.get('errorCodeVariable'),
+          errorMessageVariable = definition.get('errorMessageVariable');
+
+      if (errorCodeVariable) {
```

Comment:
The properties-panel in the latest Camunda Modeler allows to supply emty strings. Therefore "empty" variables might appear. primarily this should obviously be fixed in the properties-panel. But to make this library even more robust, we could trim whitespaces and check that it is not an empty string

![Screenshot_20210120_131829](https://user-images.githubusercontent.com/42800119/105173906-007ade00-5b22-11eb-9104-f2251816912b.png)

Reply:
Personally, I'd not see it as a part of this library to do any validation of variable names.

Reply:
Okay. 
I just don't like it because we show information which in fact is wrong.
I will create an issue in bpmn-js-properties-panel.

Reply:
But I agree that it should be fixed in bpmn-js-properties-panel => that is the right place to do it.

Reply:
=> https://github.com/bpmn-io/bpmn-js-properties-panel/issues/419

Reply:
Thanks!